app/helpers/tax_return_status_helper.rb

```diff
@@ -27,9 +27,12 @@ def self.stage_and_status_translation(status)
     "#{stage_translation_from_status(status)}/#{status_translation(status)}"
   end
 
-  def language_options
+  def language_options(only_locales: true)
     all_interview_languages = I18n.backend.translations.dig(I18n.locale, :general, :language_options)
-    all_interview_languages.select { |key, _| I18n.locale_available?(key) }.invert # only show available locales
+    if only_locales
+      return all_interview_languages.select { |key, _| I18n.locale_available?(key) }.invert
```

Comment:
When `only_locales` is `true`, the hash (?) is inverted but not sorted, like if `only_locales` is `false`. Does its also need `.sort`?

Reply:
I'm attempting to use the same helper to accomplish a slightly different task, without breaking or altering original functionality. If you take a look at the keys in the en.yml file that is the source here, the keys for the languages are in ABC order. This means that de: German is at the top, which is kind of odd when you see them in a drop down menu, so I'm sorting them. en: English and es: Spanish are already in alphabetical order and those are the only existing locales, so I didn't need to apply a sort to them in order to maintain the current functionality. There's no harm to not sorting them, but also no real harm to calling sort on an array of length two. Good catch on the asymmetry.